src/transform.css

```diff
@@ -1,90 +1,90 @@
-[class*=' tt-'],
-[class^='tt-'],
-[class*=' tb-'],
-[class^='tb-'],
-[class*=' tl-'],
-[class^='tl-'],
-[class*=' tr-'],
-[class^='tr-'] {
+[class*=' trans-t-'],
+[class^='trans-t-'],
+[class*=' trans-b-'],
+[class^='trans-b-'],
+[class*=' trans-l-'],
+[class^='trans-l-'],
+[class*=' trans-r-'],
+[class^='trans-r-'] {
   transition: transform 0.2s ease-out;
 }
 
-.tt-1:hover {
+.trans-t-1:hover {
```

Comment:
when using `trans...` you're unable to distinguish between `translate` and `transform`. how about expanding to the full word?